packages.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
@staaldraad not sure how this file got generated?

Reply:
weird, I also didn't think it got included in the branch. I'll remove it, there aren't / shouldn't be any package changes